src/main/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.java

```diff
@@ -75,23 +75,35 @@
  */
 public class XpathQueryGenerator {
 
-    /** The root ast. */
+    /**
+     * The root ast.
+     */
```

Comment:
Please revert all unrelated updates of code, you can send them in separate PR

Reply:
Done